faraday-net_http.gemspec

```diff
@@ -22,6 +22,8 @@ Gem::Specification.new do |spec|
   spec.files = Dir.glob('lib/**/*') + %w[README.md LICENSE.md]
   spec.require_paths = ['lib']
 
+  spec.add_dependency 'net-http', '~> 0.1'
```

Comment:
This needs Ruby 2.6.0. I suppose we can just demote this to a development dependency until Ruby 2.5 is no longer supported?